sympy/printing/latex.py

```diff
@@ -2393,8 +2393,12 @@ def _print_Contains(self, e):
         return r"%s \in %s" % tuple(self._print(a) for a in e.args)
 
     def _print_FourierSeries(self, s):
-        if s.an.formula is S.Zero and s.bn.formula is S.Zero:
-            return self._print(s.a0)
+        if isinstance(s.an, Dict) and isinstance(s.bn, Dict):
+            if not s.an and not s.bn:
+                return self._print(s.a0)
+        elif hasattr(s.an, 'formula') and hasattr(s.bn, 'formula'):
+            if s.an.formula is S.Zero and s.bn.formula is S.Zero:
+                return self._print(s.a0)
         return self._print_Add(s.truncate()) + r' + \ldots'
```

Comment:
This is not the proper fix. It should not be necessary to check for the attributes like this here.